src/api/search.js

```diff
@@ -89,6 +89,7 @@ class Search {
       limit,
       layers,
       country,
```

Comment:
We can also update to `countryCode` here (will accept in the endpoint now)

Reply:
ok yea i guess we can support both so nothing breaks